Wabbajack.Lib/MO2Compiler.cs

```diff
@@ -112,10 +116,12 @@ protected override bool _Begin()
                         { Path = Path.Combine(Consts.LOOTFolderFilesDir, p.RelativeTo(lootPath)) });
             }
 
+            if (cancel.IsCancellationRequested) return false;
             UpdateTracker.NextStep("Cleaning output folder");
             if (Directory.Exists(ModListOutputFolder))
                 Utils.DeleteDirectory(ModListOutputFolder);
 
+            if (cancel.IsCancellationRequested) return false;
             UpdateTracker.NextStep("Finding Install Files");
```

Comment:
This verbosity is a side effect of swapping the systems to CancellationTokens instead of thread aborts.   It's more compatible, as code that does `Task.Run()` calls cannot use the abort.  These would be seen in places like MO2InstallerVM's BeginCommand.   Eventually we'll want to add "stop installation" functionality, where we'd pass a cancellation token into the MO2Installer.Begin() call.

So I feel like this cancellation checking verbosity is just a necessary downside. 

Additionally, I think we need to extend it further, by passing the cancellation token into heavier steps, to have those steps check it periodically so they can also exit out early when appropriate.  Otherwise if cancellation is requested on a heavy step that takes 10m, it won't actually cancel for those 10 minutes if that step doesn't ever check the token.